express-hello-world/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+app.use(function (req, res, next) {
+  // console.log('Time:', Date.now());
+  next();
+  res.send('Hello');
+});
+
+app.listen(3000, () => {
+});
```

Comment:
You should always have a console log in your `listen` callback so you can verify your server started properly. You can also use the disable comment here:

```JAVASCRIPT
// eslint-disable-next-line no-console
console.log('Server listening on port 3000');
```